rollup.config.js

```diff
@@ -1,44 +1,71 @@
+/* eslint-disable header/header */
+
 import json from '@rollup/plugin-json';
 import alias from '@rollup/plugin-alias';
 import commonjs from '@rollup/plugin-commonjs';
-import resolve from '@rollup/plugin-node-resolve';
+import nodeResolve from '@rollup/plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import { terser } from 'rollup-plugin-terser';
 import babel from '@rollup/plugin-babel';
 const helpers = require('./rollup.helpers');
 
-export default {
-  input: 'src/main.js',
-  output: {
-    file: process.env.DEBUG_BUILD ? 'dist/splunk-rum.debug.js' : 'dist/splunk-rum.js',
-    format: 'iife'
+const typescriptPlugin = typescript({
+  typescript: require('typescript'),
+  useTsConfigDeclarationDir: true,
+  clean: true,
+  check: false,
+  esModuleInterop: true,
+});
+
+const babelPlugin = babel({ 
+  babelHelpers: 'bundled',
+  exclude: /node_modules/,
+});
+
+export default [
+  {
+    input: 'src/indexBrowser.js',
+    output: {
+      file: 'dist/browser/splunk-otel-web.js',
+      format: 'iife',
+      name: 'SplunkRum',
+      sourcemap: true,
```

Comment:
Our .js file sourcemaps seem to work fine but when I try to look for otel files nothing shows up.
Is it supposed to work?

Reply:
probably Typescript, I'll try to fix that

Reply:
I couldn't find a way to fix this without major structure changes